Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -241,74 +245,183 @@ private T CallPlexHelper<T>(Func<PlexHelper, T> act)
             return act(PlexHelper.GetForUser(user));
         }
 
-        #region plexapi
-        public class PlexEventSuper
-        {
-            public string user { get; set; }
-            public string Owner { get; set; }
-        }
-#pragma warning disable 0649
+        #region Plex Event Models
```

Comment:
Why change the region name?

Reply:
Because I didn't like the lower-case and concatenated name. Also, the whole class was for the API, while the region only contained the models, and the base model was the event so I tried to rename it accordingly. Anyways, reverted.